consuming/visualization/terminal_viz.py

```diff
@@ -0,0 +1,367 @@
+import logbook.configuration as log
+
+class MainScreen():
+    # Where to store them best? 7   '0123456'
+    mapIdentifiers = {'Empty':      '       ',
```

Comment:
Wouldn't it be better to move such a mapping into config and have an instance containing this strings bounded as properties? Then, you would have type check/safety at compilation time avoiding missing map entries at execution time.